device/nokia/x86_64-nokia_ixr7250e_sup-r0/platform_ndk.json

```diff
@@ -31,6 +31,38 @@
     {
       "key": "sonic_log_level",
       "stringval": "debug"
+    },
+    {
+      "key": "ungraceful_reboot_keep_sfm_alive",
+      "stringval": "yes"
+    },
+    {
+      "key": "ungraceful_reboot_log_interval",
+      "intval": 10
+    },
+    {
+      "key": "ungraceful_reboot_sfm_online",
```

Comment:
There are two similar attributes to not reset SFM on SUP restart -  ungraceful_reboot_sfm_online and ungraceful_reboot_keep_sfm_alive. Are they both for different purposes ?

Reply:
> There are two similar attributes to not reset SFM on SUP restart - ungraceful_reboot_sfm_online and ungraceful_reboot_keep_sfm_alive. Are they both for different purposes ?

Yes.   ungraceful_reboot_keep_sfm_alive is for not to powercycle the SFMs when it is a ungraceful reboot.   ungraceful_reboot_sfm_online is for allowing NDK to report SFM "online" status to PMON API query which will bring up the related  swss/syncd   
"ungraceful_reboot_keep_sfm_alive" enable/disable the ungraceful reboot support while "ungraceful_reboot_sfm_online"  bring up or keep down the related swss/syncd when Supervisor is ungraceful reboot.